code/game/objects/items/mountable_frames/trophy_mount.dm

```diff
@@ -0,0 +1,151 @@
+/obj/item/mounted/frame/trophy_mount
+	name = "trophy mount"
+	desc = "A wooden trophy mount."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "trophy_mount"
+	w_class = W_CLASS_MEDIUM
+	flags = FPRINT
+	w_type = RECYK_WOOD
+	frame_material = /obj/item/stack/sheet/wood
+	sheets_refunded = 2
+	autoignition_temperature = AUTOIGNITION_WOOD
+	var/obj/item/held_item
+	var/list/params_list
+	var/icon/clicked //Because BYOND can't give us runtime icon access, this is basically just a click catcher
+
+/obj/item/mounted/frame/trophy_mount/New()
+	..()
+	update_icon()
+
+/obj/item/mounted/frame/trophy_mount/Destroy()
+	if(held_item)
+		qdel(held_item)
```

Comment:
Is this called when the trophy mount is destroyed? If so, what is stopping someone from putting the disk in one of these and then blowing it up with a welderbomb or something?

Reply:
What's stopping them from just blowing the disk itself up with a welderbomb?
The nuke disk isn't indestructible, it just respawns when destroyed.

Reply:
Okay, I did not know that it respawned when destroyed.

Reply:
Yeah, destroying the nukedisk isn't much of a problem. Nullspacing it inside a container is the big no no.

Reply:
Even still, a while back we added a system for the nukedisk to check for and handle being nullspaced.